cdc/sink/codec/maxwell.go

```diff
@@ -256,12 +310,8 @@ func (d *MaxwellEventBatchEncoder) EncodeDDLEvent(e *model.DDLEvent) (*MQMessage
 	keyBuf := new(bytes.Buffer)
 	var versionByte [8]byte
```

Comment:
`keyLenByte`,`valueLenByte`,`versionByte` is not used, we can remove them

Reply:
and `bytes.Buffer` is not necessary

Reply:
key is not necessay,but test case need it.

Reply:
You can remove `keyLenByte`,`valueLenByte` and `versionByte` from the output. You need to keep the **key itself** in the output to conform to the official specifications.

Reply:
Can I remove decode funcation? It doesn't necessary.

Reply:
done